test/test-runner.ts

```diff
@@ -230,10 +230,42 @@ async function initializeSession(
 /**
  * run a single model test case. the inputs/outputs tensors should already been prepared.
  */
-export async function runModelTestSet(context: ModelTestContext, testCase: Test.ModelTestCase): Promise<void> {
-  Logger.verbose('TestRunner', `Start to run test data from folder: ${testCase.name}`);
+// TODO fix the reshape ops to be compatible with webgl1
```

Comment:
reshape and flatten ops?

Reply:
although flatten is implemented with reshape, it would be good to look into flatten as well for corner cases. updated.